internal/pkg/describe/lb_web_service.go

```diff
@@ -252,6 +265,47 @@ func (e envVars) humanString(w io.Writer) {
 	}
 }
 
+type Secrets struct {
+	Name        string `json:"name"`
+	Environment string `json:"environment"`
+	ValueFrom   string `json:"valueFrom"`
+}
+
+type secrets []*Secrets
+
+func (s secrets) humanString(w io.Writer) {
+	fmt.Fprintf(w, "  %s\t%s\t%s\n", "Name", "Environment", "Value From")
+	fmt.Fprintf(w, "  %s\t%s\t%s\n", "----", "-----------", "----------")
+	var prevName string
+	var prevValueFrom string
+	for _, secret := range s {
+		// If the valueFrom is not an ARN, preface it with "parameter/"
+		var outputValueFrom string
+		_, err := arn.Parse(secret.ValueFrom)
+		if err != nil {
+			outputValueFrom = fmt.Sprintf("parameter/%s", secret.ValueFrom)
+		} else {
+			outputValueFrom = secret.ValueFrom
+		}
+		// Instead of re-writing the same secret valueFrom, we replace it with "-" to reduce text.
+		if secret.Name != prevName {
```

Comment:
@huanjani @iamhopaul123 ahh this only works because we sort the secrets outside of this method by environment first and name.
Mmm I wonder if it would safer if the sorting was moved inside these methods.

Reply:
Is there any reason why they would need to be sorted for JSON? If not, yeah, I'll move the sorting.

Reply:
I don't think so! they can be out of order for JSON

Reply:
but how about testing? they'll be unstable unless we only have one item in the JSON

Reply:
im more inclined to keep it sorted because it also looks nicer i think.

Reply:
> im more inclined to keep it sorted because it also looks nicer i think.

the json? but no other fields are sorted, are they?

Reply:
> but how about testing? they'll be unstable unless we only have one item in the JSON

unit tests are passing; are you thinking about integ/e2e?

Reply:
oh sorry i misunderstood. I mean i want the content we display to users are ordered.